src/map/itemdb.cpp

```diff
@@ -177,13 +160,9 @@ uint64 ItemDatabase::parseBodyNode(const YAML::Node &node) {
 
 		item->value_sell = sell;
```

Comment:
```suggestion
		item->value_sell = sell;
		item->value_buy = 0;
```
Define `sell` should 'remove' the current `buy`